modules/serverless-create-inmemorychannel-YAML.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-create-inmemorychannel-YAML_{context}"]
+= Creating a development channel using YAML
+
+You can create a channel by using YAML with the cluster default configuration, as shown in the following procedure.
+
+.Prerequisites
+To create channels ensure that:
+
+* The {ServerlessOperatorName}, Knative Serving and Knative Eventing are installed on your {product-title} cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+.Procedure
+
+. Create a `Channel` object.
+.. Create a YAML file and copy the following sample code into it:
++
+
```

Comment:
Just a stylistic thing, but I'd remove the blank line here between the `+` and following YAML snippet for consistency.

Reply:
It's actually part of the guidelines to always have a blank line before the code blocks afaik https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Reply:
I believe that only applies when the code block is not in an ordered list.

![image](https://user-images.githubusercontent.com/1738558/95497226-ffa5ae00-096f-11eb-8b9d-b06154176833.png)

Most certainly not a big deal, just wanted to call it out. If the Serverless docs do it this way throughout, it's probably better to stay consistent with the rest anyways. Just figured I'd mention it. :wink:

Reply:
Thanks @codyhoag - I've been leaving a space between them so I'll leave as is for now to keep it consistent for Serverless (it renders the same way anyway), but next time I'm doing any style updates, e.g. when I'm reviewing docs for the new style stuff that Andrea has been working on, I'll update these. Thanks for clarifying this.